circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  python:
+    version: 2.7.9
+  environment:
+    SKIP_CLEANUP: true
+    # comma separated list
+    CIRCLECI_FLAVORS: docker
+
+dependencies:
+  pre:
```

Comment:
You probably want to override this phase, as otherwise `python setup.py install` and others are run, which could lead to unwanted effects.